tests/components/mqtt/test_server.py

```diff
@@ -18,14 +18,12 @@ def teardown_method(self, method):
         """Stop everything that was started."""
         self.hass.stop()
 
-    @patch('passlib.apps.custom_app_context', return_value='')
-    @patch('tempfile.NamedTemporaryFile', return_value=MagicMock())
+    @patch('passlib.apps.custom_app_context', Mock(return_value=''))
```

Comment:
Interesting, I didn't realize that providing a Mock will prevent a function argument to be added 👍